cmd/test.go

```diff
@@ -83,19 +85,23 @@ func validateArgs() error {
 	return nil
 }
 
-func RunTests() []*unversioned.FullResult {
-	fullResults := []*unversioned.FullResult{}
+func RunTests(out *output.OutWriter) bool {
+	pass := true
 	for _, file := range configFiles {
+		out.Banner(file)
 		tests, err := Parse(file)
 		if err != nil {
 			logrus.Fatalf("Error parsing config file: %s", err)
 		}
-		fullResults = append(fullResults, &unversioned.FullResult{
-			FileName: file,
-			Results:  tests.RunAll(),
-		})
+		results := tests.RunAll(out)
+		for _, result := range results {
```

Comment:
You dont need to loop over results here. 
You are already doing it in out.FinalResults(results) and returning a bool.

Reply:
yep, thanks